src/resolver.cpp

```diff
@@ -235,211 +222,95 @@ std::string UsdOpenAssetIOResolver::_CreateIdentifier(
-           << "\n  assetPath: " << assetPath;
-    logger_->debug(logMsg.str());
-  }
-
-  auto result = catchAndLogExceptions(
-      [&] {
-        // TODO(DF): Give the manager a chance to provide the file extension.
-        return ArDefaultResolver::_GetExtension(
-            locationInManagerContextForEntity(manager_, readContext_, assetPath));
-      },
-      logger_, fnName);
-
-  {
-    std::ostringstream logMsg;
-    logMsg << "[tid=" << std::this_thread::get_id() << "] "
-           << "result: " << result;
-    logger_->debug(logMsg.str());
-  }
-
-  return result;
+  return ArDefaultResolver::_GetExtension(assetPath);
 }
 
 ArAssetInfo UsdOpenAssetIOResolver::_GetAssetInfo(const std::string &assetPath,
                                                   const ArResolvedPath &resolvedPath) const {
-  const std::string fnName = TF_FUNC_NAME();
-  {
-    std::ostringstream logMsg;
-    logMsg << "[tid=" << std::this_thread::get_id() << "] " << fnName
-           << "\n  assetPath: " << assetPath
-           << "\n  resolvedPath: " << resolvedPath.GetPathString();
-    logger_->debug(logMsg.str());
-  }
-  // TODO(DF): Determine if there is value in supporting this via the
-  //  manager, including any useful data that can be stuffed into the
-  //  generic `resolverInfo` VtValue member.
-  auto result = ArDefaultResolver::_GetAssetInfo(assetPath, resolvedPath);
-
-  {
-    std::ostringstream logMsg;
-    logMsg << "[tid=" << std::this_thread::get_id() << "] "
-           << "\n  result.assetName: " << result.assetName
-           << "\n  result.repoPath: " << result.repoPath;
-    logger_->debug(logMsg.str());
-  }
-
-  return result;
+  return ArDefaultResolver::_GetAssetInfo(assetPath, resolvedPath);
 }
 
 ArTimestamp UsdOpenAssetIOResolver::_GetModificationTimestamp(
     const std::string &assetPath, const ArResolvedPath &resolvedPath) const {
-  const std::string fnName = TF_FUNC_NAME();
-  {
-    std::ostringstream logMsg;
-    logMsg << "[tid=" << std::this_thread::get_id() << "] " << fnName
-           << "\n  assetPath: " << assetPath
-           << "\n  resolvedPath: " << resolvedPath.GetPathString();
-    logger_->debug(logMsg.str());
-  }
-
-  auto result = catchAndLogExceptions(
-      [&] {
-        if (manager_->isEntityReferenceString(assetPath)) {
-          // Deliberately use a valid fixed timestamp, which will force caching
-          // until we implement a more considered method.
-          // TODO(DF): Consider how best to handle this via OpenAssetIO.
-          return ArTimestamp{0};
-        }
-        return ArDefaultResolver::_GetModificationTimestamp(assetPath, resolvedPath);
-      },
-      logger_, fnName);
-
-  {
-    std::ostringstream logMsg;
-    logMsg << "[tid=" << std::this_thread::get_id() << "] "
-           << "result: " << result.GetTime();
-    logger_->debug(logMsg.str());
-  }
-
-  return result;
+  return ArDefaultResolver::_GetModificationTimestamp(assetPath, resolvedPath);
```

Comment:
[question] We've lost the ability for the manager to report a different modification time than what's on disk.  But practically speaking, there's no reason for it to want to do that, right? - if it wants to trigger a re-read of data, then it would return a different path from `_Resolve`.